<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>359</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>22</x>
      <y>12</y>
      <width>361</width>
      <height>221</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QPushButton" name="pushButton">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>106</height>
         </size>
        </property>
        <property name="text">
         <string>Пуск</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_2">
        <property name="text">
         <string>Записать скорость</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit">
        <property name="maxLength">
         <number>3</number>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Гц</string>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="listWidget"/>
     </item>
    </layout>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>240</y>
      <width>331</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Измерение на нескольких частотах</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QSplitter" name="splitter_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>260</y>
      <width>346</width>
      <height>80</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QSplitter" name="splitter_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Стартовая частота:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Финишная частота:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Шаг частоты:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Пауза между измерениями:</string>
      </property>
     </widget>
    </widget>
    <widget class="QSplitter" name="splitter_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QLineEdit" name="EditStart">
      <property name="text">
       <string>1</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="EditStop">
      <property name="text">
       <string>30</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="EditStep">
      <property name="text">
       <string>1</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="EditSleep">
      <property name="text">
       <string>30</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="ButtonStop">
     <property name="text">
      <string>СТОП</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>120</y>
      <width>21</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
 <slots>
  <slot>AddTime()</slot>
 </slots>
</ui>
